ql/src/test/queries/clientpositive/groupingset_optimize_hive_28489.q

```diff
@@ -0,0 +1,26 @@
+create table grp_set_test (key string, value string, col0 int, col1 int, col2 int, col3 int);
```

Comment:
We may test this feature with both EXPLAIN and non-EXPLAIN. That's because all of relevant test cases, i.e. `groupingset_optimize_hive_28489.q` and TPC-DS, run only EXPLAIN.

Reply:
Added two identical select commands that run with and without optimization.